townService/src/town/games/ChessGame.ts

```diff
@@ -35,6 +35,8 @@ export default class ChessGame extends Game<ChessGameState, ChessMove> {
       moves: [],
       status: 'WAITING_TO_START',
       halfMoves: 0,
+      capturedBlackPieces: [],
```

Comment:
If we add this to the constructor will we error? Due to it extending ChessGameState